lms/templates/fields/field_text_account.underscore

```diff
@@ -4,5 +4,5 @@
 </div>
 <span class="u-field-message" id="u-field-message-<%- id %>">
     <span class="u-field-message-notification" aria-live="polite"></span>
-    <span class="u-field-message-help" id="u-field-message-help-<%- id %>"> <%- message %></span>
+    <span class="u-field-message-help" id="u-field-message-help-<%- id %>"><%= HtmlUtils.ensureHtml(message) %></span>
```

Comment:
@andy-armstrong: As mentioned earlier, I needed this to not escape the bold tag.  I squashed so just wanted to highlight this change.